### PR TITLE
Add tests for boot_id and uuid procfs entries

### PR DESCRIPTION
**Pull request by pekdon**
This is a follow up after merge of OS-8280, adding a couple of simple functional tests of the kernel random entries in procfs.

### CODE REVIEW

**Comment by danmcd** [comment]
Have you run these tests both in pre-your-fix and post-your-fix environments? I'm good to squash with documented testing.

File: src/procfs.c [hunk 1]
```diff
@@ -292,5 +357,15 @@ main(int argc, char **argv)
 		return (test_fail(TST_NAME, "procfs 6 - writable files"));
 	}
 
+	/* Test 7 - /proc/sys/kernel/random/boot_id */
+	if (check_kernal_random_boot_id() != 0)  {
```
Comment (danmcd) [inline src/procfs.c]: Ooops, spelling mistake:  "kernel", not "kernal".  (Are you an old Amiga person?)

Comment (pekdon) [reply src/procfs.c]: No, just a combination of a typo and then dabbrev-expand I guess. (Atari, not Amiga). Will fix.

**Review by danmcd — APPROVED** [review]
Please document you've run this test on before-and-after your procfs fixes and I'll finish the pull.

**Comment by pekdon** [comment]
I've now run the tests on:

* SmartOS pre OS-8280, boot_id and uuid tests fail (Debian 9 LX).
* SmartOS post OS-8280, boot_id and uuid tests PASS (Debian 9 LX).
* Native Linux, boot_id and uuid tests PASS (Debian Linux 5.10.19)

**Pull request merged**